examples/jsm/loaders/GLTFLoader.js

```diff
@@ -1479,6 +1480,12 @@ var GLTFLoader = ( function () {
 		this.textureLoader = new TextureLoader( this.options.manager );
 		this.textureLoader.setCrossOrigin( this.options.crossOrigin );
 
+		// Use an ImageBitmapLoader if imageBitmaps are supported. Moves much of the
+		// expensive work of uploading a texture to the GPU off the main thread.
+		if ( typeof createImageBitmap !== 'undefined' ) {
+			this.textureLoader.setImageLoader( new ImageBitmapLoader( this.options.manager ) );
+		}
```

Comment:
Note that we still need to support the `examples/js/loaders/GLTFLoader.js` version of the loader until EOY 2020 — this file is autogenerated from it. Could you update that copy, then run `node utils/modularize.js` to regenerate the `examples/jsm/*` copy?

The change itself looks good to me!

Reply:
Wasn't aware of that, thanks! It's fixed now.